docs/sighup.md

```diff
@@ -32,3 +32,9 @@ comparing paths from `shard.blobstor` section. After this we have 3 sets:
 | Changed section | Actions                                                                                                              |
 |-----------------|----------------------------------------------------------------------------------------------------------------------|
 | `path`          | If `path` is different, metabase is closed and opened with a new path. All other configuration will also be updated. |
+
+### Morph
+
+| Changed section | Actions                                                                                                                                                                                                |
+|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `endpoints`     | Updates N3 endpoints.<br/>If in the `endpoints` there is no endpoint that the client is connected to, it will attempt to connect to another endpoint N3. Node service can be interrupted in this case. |
```

Comment:
If new `endpoints` do not contain the endpoint client is connected to, it will reconnect to another endpoint from the new list

Smth like that (other sentences are ok)